pulsar-client-tools/src/main/java/org/apache/pulsar/shell/ConfigShell.java

```diff
@@ -256,6 +257,9 @@ public boolean run() {
     @Parameters(commandDescription = "Create a new configuration.")
     private class CmdConfigCreate extends CmdConfigPut {
 
+        @Parameter(description = "Configuration name", required = true)
+        protected String name;
```

Comment:
Is this somewhat of a breaking change since we already release Pulsar Shell in 2.11.0?

That is, previously, users could create config without specifying a name.

Reply:
if you look closely I just moved it in the child class because the create and update commands must have different autocompletion. at "runtime" it's the same and the syntax isn't changed

Reply:
Thanks for your explanation! Clone the code and confirm.